client/devdocs/design/app-components.jsx

```diff
@@ -0,0 +1,155 @@
+/**
+* External dependencies
+*/
+var React = require( 'react' ),
+	page = require( 'page' ),
+	toTitleCase = require( 'to-title-case' ),
+	trim = require( 'lodash/string/trim' );
+
+/**
+ * Internal dependencies
+ */
+var SearchCard = require( 'components/search-card' ),
+	CommentButtons = require( 'components/comment-button/docs/example' ),
+	LikeButtons = require( 'components/like-button/docs/example' ),
+	FollowButtons = require( 'components/follow-button/docs/example' ),
+	Sites = require( 'lib/sites-list/docs/example' ),
+	SitesDropdown = require( 'components/sites-dropdown/docs/example' ),
+	Theme = require( 'components/theme/docs/example' ),
+	PostSchedule = require( 'components/post-schedule/docs/example' ),
+	HeaderCake = require( 'components/header-cake' ),
+	Collection,
+	FilterSummary,
+	Hider;
+
+Hider = React.createClass( {
```

Comment:
Yes, all of those. And remove as much of the boilerplate currently needed to add something there.